### ISSUE TITLE
not created folder bazel-bin on apollo (branch master)

### ISSUE THREAD

**Issue by AlexandrZabolotny** [ApolloAuto/apollo#11220]
Why do not create folder bazel-bin with library??
There is a need to install apollo again before a new launch

### System information

- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
- **ros kinetic**
- **Apollo installed from (source)**:
- **Apollo version (5.5.0)**:

**Comment by changsh726** [comment]
The bazel-bin folder will be created by bazel automatically if you follow the build steps, could you please discribe your problem in more detail?  

> Why do not create folder bazel-bin with library??
> There is a need to install apollo again before a new launch
> 
> ### System information
> * **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
> * **ros kinetic**
> * **Apollo installed from (source)**:
> * **Apollo version (5.5.0)**:

The bazel-bin folder will be created by bazel automatically if you follow the build steps, could you please discribe your problem in more detail?

**Comment by changsh726** [comment]
> Why do not create folder bazel-bin with library??
> There is a need to install apollo again before a new launch
> 
> ### System information
> * **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
> * **ros kinetic**
> * **Apollo installed from (source)**:
> * **Apollo version (5.5.0)**:

The bazel-bin folder will be created by bazel automatically if you follow the build steps. Could you please discribe your problem in more detail?

**Comment by AlexandrZabolotny** [comment]
**My steps**  
1.  
git clone git@github.com:ApolloAuto/apollo.git  
cd apollo  
git checkout master  
2. bash docker/scripts/dev_start.sh  
3. bash docker/scripts/dev_into.sh  
4. bash apollo.sh build  

**after finish installation was successful**  
And result in my folder  

![apollo](https://user-images.githubusercontent.com/61351599/81660212-230d1100-9443-11ea-82e9-b7cb29f213d4.png)  

**My problem -  bazel-bin folder was not created by bazel automatically**

**Comment by changsh726** [comment]
The bazel-bin in the original folder is a soft link, you can visit the bazel-bin folder in the docker container.

**Comment by AlexandrZabolotny** [comment]
**Why after reboot my PC and start apollo I see this?**

```
apollo@in_dev_docker:/apollo$ bash ./scripts/bootstrap.sh 
nohup: appending output to 'nohup.out'
Could not launch module monitor. Is it already built?
nohup: appending output to 'nohup.out'
Launched module dreamview.
Failed to start Dreamview. Please check /apollo/data/log or /apollo/data/core for more information
```

**Comment by changsh726** [comment]
Have you run `bash apollo.sh build` in the docker container? The build steps need to be redone after a reboot.

**Comment by AlexandrZabolotny** [comment]
https://github.com/ApolloAuto/apollo/issues/11220#issuecomment-627220444  

Did I understand correctly?
Should I rebuild the project after each reboot of the PS? Why?

But for version 5.0.0 there wasn’t

**Comment by AlexandrZabolotny** [comment]
> Have you run `bash apollo.sh build` in the docker container? The build steps need to be redone after a reboot.

Did I understand correctly?  
Should I rebuild the project after each reboot of the PS? Why?  

But for version 5.0.0 there wasn’t

**Comment by storypku** [comment]
> > Have you run `bash apollo.sh build` in the docker container? The build steps need to be redone after a reboot.
> 
> Did I understand correctly?
> Should I rebuild the project after each reboot of the PS? Why?
> 
> But for version 5.0.0 there wasn’t

Try the lastest master branch, IIRC this issue was fixed in pull#11215

**Comment by AlexandrZabolotny** [comment]
thank
